cdn_lambda/functions/origin_response/origin_response.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import re
+
+LOG = logging.getLogger("origin-response")
+
+
+def lambda_handler(event, context):
+    # pylint: disable=unused-argument
+
+    request = event["Records"][0]["cf"]["request"]
+    original_uri = request["headers"]["original-uri"][0]["value"]
```

Comment:
Could this please be written in such a way that it won't crash if the header is absent?  It would be nice to keep the lambda able to execute without crashing when passed a "standard" CloudFront event, and not have it 100% rely on custom fields we've added.